release notes/v0.38.0.md

```diff
@@ -0,0 +1,237 @@
+      iterations: 30,
+      thresholds: {
+          my_counter: ['p(95)<200'], // Counter metrics do not support the p() aggregation method
+      },
+    };
+    ```
+
+### Support for encrypted TLS private keys
+
+It is now possible to use passphrase protected private keys when performing TLS authentication. Using the `password` property of an options' `tlsAuth` object, you can now indicate to k6 the passphrase to use to decrypt a private key. 
+
+```javascript
+export const options = {
+  tlsAuth: [
+    {
+      domains: ['example.com'],
+      cert: open('mycert.pem'),
+      key: open('mycert-key.pem'),
+      password: 'mycert-passphrase',
+    },
+  ],
+};
+```
+
+Thanks, @Gabrielopesantos, for the [contribution](https://github.com/grafana/k6/pull/2488).
+
+### Improve JSON output's performance ([2436](https://github.com/grafana/k6/pull/2436))
+
+The JSON output was optimized and now should around 2x as more performant at outputting metrics.  This means that it either will be able to export twice as many metrics and/or to use half the resources to do the same amount of metrics. As a side effect, there is a slight breaking change in which the `tags` field is no longer sorted, while it was previously.
+
+### Treat panics as interrupt errors ([2453](https://github.com/grafana/k6/pull/2453))
+
+We changed the behavior of how k6 treats panic, which can happen during the execution of JavaScript test scripts code. Previously, the behavior was to catch the panic and log it as an error. Starting with **v0.38.0**, whenever k6 observes a panic, it logs an error like before, but more importantly, it will abort the script execution. This change should help identify and address the issues earlier.
+
+### Miscellaneous
+
+* To achieve consistency with the existing `--log-output` command-line flag, the `--logformat` flag is now considered deprecated and the new `--log-format` takes its place ([2410](https://github.com/grafana/k6/pull/2410)). The same goes for the `K6_LOGFORMAT` environment variable, which is now deprecated and replaced by `K6_LOG_FORMAT` to be consistent with the existing `K6_LOG_OUTPUT`.
+* [2411](https://github.com/grafana/k6/pull/2411) The k6 command-line UI (logo, test description and progerss bars) can now effectively be disabled using the [`--quiet`](https://k6.io/docs/using-k6/options/#quiet) flag.
+* [2411](https://github.com/grafana/k6/pull/2411) The logs can now be saved to a file, or sent to Loki, or completely disabled with [`--log-output`](https://k6.io/docs/using-k6/options/#log-output) flag.
+* [2411](https://github.com/grafana/k6/pull/2411) the end-of-test summary can be sent to a remote server or to file through [`handleSummary()`](https://k6.io/docs/results-visualization/end-of-test-summary/#handlesummary-callback).
+* [2429](https://github.com/grafana/k6/pull/2429) `lib/types` now exposes the source of the `NullHostnameTrie` to simplify the access to an original list of the hostnames.
+
+## Bugs Fixed!
```

Comment:
We should probably add https://github.com/grafana/k6/pull/2502 under "Known issues", given that we know about it, but don't want to fix it just before a release, given it's very small and UX-only impact

Reply:
I've added a "Known issues" section, where I tried to detail the #2502 issue. Please let me know if you think my description of the issue was accurate enough 🤝